sl_model_mod.f90

```diff
@@ -1296,6 +1467,28 @@ subroutine sl_solver(itersl, iter, dtime, starttime, mali_iceload, mali_mask, sl
       ! converged beta function at the current timestpe
       call write_sl(beta, 'beta', outputfolder, suffix=numstr)
 
+      ! output the ocean areas
```

Comment:
All of these ocean area and GMSL calculations look the same as above for the sl_init routine.  If they are identical, you could move it all to a subroutine that is called from both places.  If you'd rather leave it as you have it, that's fine too.

Reply:
At the moment, the routine `write_txt` in `sl_init_mod.f90` only supports writing data out to a new file (i.e. `status = 'replace'`). It would be great to have a separate PR focusing on improving the `i/o routine` that addresses the i/o-related issues including this one, but I would prefer to leave this as I have it for this PR.